Azure/communication-ui-library-ios
# [BugFix] Listening for new clean up action

## Purpose
New `CleanupCallStates` action that will reset the the recording and transcript value. 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
User A and User B join teams call
User A begins recording and transcription
User B loses internet and is kicked out into the set up view
User A stops the recording and transcription
User B turns on internet and rejoins call
Verify the banner is not showing in call

## Other Information
